1st Revision

TetsujinXLIV opened the pull request:
Added some lines. It not fills the 8X8 matrix and then will print out
the matrix. Changed some formatting for presentation purposes.

TetsujinXLIV committed:
1st Revision

Added some lines. It not fills the 8X8 matrix and then will print out
the matrix. Changed some formatting for presentation purposes.

TetsujinXLIV committed:
Newest Additions

Added the h file. Changed height and width to just size. added a bool
function to be the recursive function that does the moves all 64 times.

TetsujinXLIV committed:
Okay now this is most current

Same changes from the last one they just werent saved yet

TetsujinXLIV committed:
Adds Backtrack checker

kcnibungco committed:
random number generator

kcnibungco committed:
next position = initial + KTMOV

TetsujinXLIV committed:
Bool Tour changes

added if statement that checks if the tour has gone to all spaces on the
board and if it has then it returns true and goes to the knightstour
function

drmoreno2494 commented on KnightsTour.cpp:
```diff
@@ -0,0 +1,112 @@
+		Nextj = j + KTMOV2[a];
+
+
+	if ((i >= 0 && i <= width) && (j >= 0 && j <= height) && cb[i][j] == 0)
```
Checks if it went to all spaces on the board

kcnibungco committed:
For loops

TetsujinXLIV committed:
Kinda Working version

Removed for loop in the possibility checker, changed some of the if
statements in knights tour

TetsujinXLIV committed:
New Changes

Just changed a few values to see if it is working
